### ISSUE TITLE
Low-res static map masking

### ISSUE THREAD

**Issue by sevakon** [iarai/NeurIPS2021-traffic4cast#4]
Hi! 

Thank you for the provided data and the competition in general. 

I was working on my pipeline and considering adding masked MSE metric into my validation part, however, when performing some sanity checks I encountered the following inconsistency:

I was assuming that when multiplying the ground truth maps of the 6 frames we are predicting with the mask of the provided low-resolution static map, the number of non-zeros in the resulting tensor shouldn't decrease compared to beforehand. Yet the result was the contrary, meaning that in the ground truth labeling, some non-zero elements lie outside the provided low-resolution static map. 

Please see the following code snippet for reproducing:

```python
from data.dataset.dataset import T4CDataset
from baselines.unet import UNetTransfomer
from metrics.masking import get_static_mask

from functools import partial

import numpy as np
import torch

BASE_FOLDER = ...
CITY = "CHICAGO"

dataset = T4CDataset(
    BASE_FOLDER, f"{CITY}/training/2019*8ch.h5",
    transform=partial(
        UNetTransfomer.unet_pre_transform,
        stack_channels_on_time=True,
        zeropad2d=None, batch_dim=False)
)

city_static_map = get_static_mask(CITY, BASE_FOLDER)
city_static_map_torch = torch.from_numpy(city_static_map[:, :, 0])

for index in range(len(dataset)):
    inp_frames, out_frames = dataset[index]

    nonzero_frames = torch.count_nonzero(out_frames)
    nonzero_frames_masked = torch.count_nonzero(out_frames * city_static_map_torch)

    print(nonzero_frames)
    print(nonzero_frames_masked)

    assert torch.allclose(nonzero_frames, nonzero_frames_masked)

```

Is this the expected behavior? Should we trust or use the low-resolution static map then?

**Comment by chenkins** [comment]
Hi @sevakon 
Thanks for your question. Yes, this is expected: The low-res static map is built from a (pixeled) picture (the high-res static map) of the road map and GPS probes may also be recorded outside of this projection of the road graph (for instance while cars are on ferry lines or if the road is near the edge of the grid cells and the signal is recorded in the neighboring cell). See https://www.iarai.ac.at/traffic4cast/forums/topic/the-metric-mse/ for details.
